interactive_ai/supported_models/geti_supported_models/manifests/semantic_segmentation/base.yaml

```
@@ -0,0 +1,12 @@
+task: semantic_segmentation
+capabilities:
+  tiling: true
+
+hyperparameters:
+  evaluation:
+    metric: Dice
```

```suggestion
    metric: dice
```
all lowercase for consistency

---

[Done](https://github.com/open-edge-platform/geti/pull/547/commits/625e8e7fb3335e75d30ff4376abe3b5f5ff22709)